kale/embed/decomposition.py

```diff
@@ -14,8 +14,12 @@
 import warnings
```

Comment:
This comment is about line 1 to 12. Since you think renaming it to decomposition is better, the top docstring should be updated accordingly to specify the reasoning, i.e. what should go here or what is expected here.